documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -14,6 +14,8 @@
   [detect.excluded.directories.defaults.disabled](properties/configuration/paths.md#detect-excluded-directories-defaults-disabled-advanced) to disable these defaults.
 * Support for Gradle is now extended to Gradle 8.2.
 * Support for GoLang is now extended to Go 1.20.4.
+* Support for npm is now extended to npm 9.8.1.
+  Support for npm workspaces.
 * Removed reliance on [solution_name] libraries for init-detect.gradle script to prevent them from being included in the Gradle dependency verification of target projects.
```

Comment:
While you are in here, would you mind moving "_Removed reliance on [solution_name] libraries for init-detect.gradle script to prevent them from being included in the Gradle dependency verification of target projects._" above the first "Support for..." entry? Just to group the new "Support" items together.